config/static-deployment/deployment.yaml.tpl

```diff
@@ -26,6 +26,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: PKO_SENTRY_DSN
+          value: https://2a9bea4b2ee04fafa940ee32b0e4660b@glitchtip.stage.devshift.net/17
```

Comment:
@thetechnick I am not sure that we should be applying the DNS directly like this ?